siddhi-io/src/main/resources/archetype-resources/component/src/main/java/source/__classNameOfSource__.java

```diff
@@ -80,8 +80,7 @@
                 )
         }
 )
-
-// for more information refer https://wso2.github.io/siddhi/documentation/siddhi-4.0/#sources
+// for more information refer https://siddhi-io.github.io/siddhi/documentation/siddhi-4.x/query-guide-4.x/#source
```

Comment:
Shouldn't we refer to Siddhi 5.x here?

Reply:
I have still not pushed the changes for 5.x., these changes are applied for 4.x.x.